mmdet3d/ops/spconv/overwrite_spconv/write_spconv2.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import itertools
+
+from mmcv.cnn.bricks.registry import CONV_LAYERS
+from torch.nn.parameter import Parameter
+
+
+def register_spconv2():
```

Comment:
Adding some docstring for explaining that Once spconv2.0 can be used, we need to register the spconv2.0 to overwrite the original spconv in mmcv.